great-idea/css/index.css

```diff
@@ -63,4 +63,242 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
```

Comment:
All this css looks awesome. Your use of flexbox is spot on. The organization of your selectors is logical and flows nicely. You use classes thoughtfully to keep styling succinct.